### PR TITLE
Documentation for MDL-80396

### PR DESCRIPTION
**Pull request by roland04**

### CODE REVIEW

**Comment by andrewnicols** [comment]
Is this a dupe of #848?

**Comment by andrewnicols** [comment]
Converting to draft to prevent accidental merge as the linked issues are not ready

File: docs/devupdate.md [hunk 1]
```diff
@@ -40,6 +40,44 @@ will only update the second enrol instance.
 
 A new method `enrol_plugin:find_instance()` is added to the enrol plugin interface to allow plugins to find an existing instance of the same type in the course. If you want your enrolment method to be supported in CSV course upload, you need to implement this [method](./apis/plugintypes/enrol#enrol_pluginfind_instance-stdclass)
 
+## Bootstrap preparations for version 5
+
+<Since version="4.4" issueNumber="MDL-71979" />
+
+Some of the Bootstrap 4 classes will be deprecated or dropped in its version 5. To prepare for this, some of the current Bootstrap 4 classes usages have been replaced with version 5 compatible classes. This will help us to upgrade to Bootstrap 5 in the future.
+
+- The .media component has been replaced with utility classes.
```
Comment (sarjona) [inline docs/devupdate.md:49]:
```suggestion
- The `.media` component has been replaced with utility classes.
```

Comment (roland04) [reply docs/devupdate.md]: Thanks a lot Sara! Changes done, I missed that.

**Review by sarjona — APPROVED** [review]
Thanks a lot for improving the devdocs. I'm happy to approve this patch (and I'll merge after rolling the weekly).

**Pull request merged**